src/components/patientsList/List.js

```diff
@@ -1,33 +1,32 @@
 import React from "react";
 import { FlatList } from "react-native";
 import PropTypes from "prop-types";
-import PatientsListItem from "./PatientListItem";
-import { PatientsContext } from "../../modules/context/PatientsContext";
-
-const PatientsList = ({ navigation }) => {
-  const { patients } = React.useContext(PatientsContext);
+import Item from "./item";
+import Patient from "../../constants/propTypes";
 
+const List = ({ navigation, patients }) => {
```

Comment:
Tutaj nie rozumiem zmiany z pobierania pacjentów z useContext na pobieranie pacjentów z propsów - to jest lepsze rozwiązanie?

Reply:
Możliwe, że przekazywanie tak docelowo dużego propsa, będzie wywoływać jakieś problemy wydajnościowe. Sprawdzę co jest lepsze.

Reply:
Propsy przekazywane są przez referencje i nie ma problemu z wydajnością. 
Nie użyłem kontekstu, bo nie potrzebuję wszystkich pacjentów, a tylko tych przefiltrowanych.